.universum.py

```diff
@@ -2,7 +2,7 @@
 
 import os
 import sys
-from universum.configuration_support import Variations
+from universum.configuration_support import Variations, Step
```

Comment:
This change might not be fitting in this PR, but it's fairly small and simple. It seems I missed some internal usages of old config format.